src/RoslynInsertionTool/scripts/HelperFunctions.ps1

```diff
@@ -37,11 +37,19 @@ function GetComponentGitHubRepoName([string] $componentGitHubRepoName) {
     }
 }
 
-function GetComponentUserName([string] $componentAzdoUri) {
-    if (IsDefaultValue $componentAzdoUri) {
+function GetComponentUserName([string] $componentUserName) {
+    if (IsDefaultValue $componentUserName) {
+        return "/cbu=vslsnap@microsoft.com"
```

Comment:
I will have a followup pass after everything is working to entirely remove vslsnap if dotnet bot is working fine.